README.md

```diff
@@ -129,7 +129,7 @@ Please see the [Contribution Guide](./CONTRIBUTING.md).
 ----------------------------------------------------------------
 > Copyright (c) 2014-2019 syuilo
 
-Misskey is open-source software licensed under the [GNU AGPLv3](LICENSE).
+Misskey is an open-source software licensed under the [GNU AGPLv3](LICENSE).
```

Comment:
The original text here is okay. "a[n] software" is grammatically incorrect.

Reply:
Thanks!

(For reference: https://forum.wordreference.com/threads/software-vs-a-software.3216740/)